docker-compose.yml

```diff
@@ -109,6 +109,8 @@ services:
       TOKENS_CONFIG: API_SERVICE_API_KEY:hex,AUTH_RESINOS_REGISTRY_CODE:hex,COOKIE_SESSION_SECRET:hex,JSON_WEB_TOKEN_SECRET:hex,MIXPANEL_TOKEN:hex,SUPERUSER_PASSWORD:hex,TOKEN_AUTH_BUILDER_TOKEN:hex,VPN_GUEST_API_KEY:hex,VPN_SERVICE_API_KEY:hex,API_VPN_SERVICE_API_KEY:API_SERVICE_API_KEY,REGISTRY2_TOKEN:TOKEN_AUTH_BUILDER_TOKEN
       TRUST_PROXY: 172.16.0.0/12
       VPN_PORT: 443
+      WEBRESOURCES_S3_BUCKET: web-resources
+      WEBRESOURCES_S3_REGION: local
```

Comment:
@ab77 I think this needs to be us-east-1 as it is the default region minio will use

Reply:
@otaviojacobi how does this region setting affect a local installation of openBalena? Or is there some defaults/hard-coding goingf on which won't work without it being explicitly set to `us-east-1` regardless of actual location?

Reply:
@ab77 This is basically a mandatory parameter to use aws sdk v3 https://github.com/aws/aws-sdk-js-v3/issues/5564#issuecomment-1852890956

Minio will use us-east-1 as default regardless of where it is actually deployed so it can work with aws configurations, therefore we use us-east-1 (as it is the default one used by minio)

Reply:
So if we omit `WEBRESOURCES_S3_REGION`, the default value will be used anyway?

Reply:
this is what I get against the open API:

```
id=$(( RANDOM ))

curl -vi "https://api.$DNS_TLD/resin/release_asset" \
  --header "Authorization: Bearer $(cat <~/.balena/token)" \
  --form 'asset=@"balena.png"' \
  --form "release=\"$id\"" \
  --form 'release asset key="balena.png"'

...
> POST /resin/release_asset HTTP/2
> Host: api.foo.balena-dev.com
> User-Agent: curl/8.5.0
> Accept: */*
> Authorization: Bearer eyJhbG...
> Content-Length: 40608
> Content-Type: multipart/form-data; boundary=------------------------6BlxEVl8hBfWNTBIO9ICrk
> 
* We are completely uploaded and fine
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
< HTTP/2 404 
HTTP/2 404 
< date: Thu, 16 May 2024 20:24:57 GMT
date: Thu, 16 May 2024 20:24:57 GMT
< content-length: 0
content-length: 0
```